tools/ci/tc/decision.py

```diff
@@ -74,6 +74,17 @@ def get_run_jobs(event):
     path_jobs = jobs.get_jobs(paths)
     all_jobs = path_jobs | get_extra_jobs(event)
     logger.info("Including jobs:\n * %s" % "\n * ".join(all_jobs))
+
+    # Exclude browser stability tests for exports from that specific browser.
+    try:
+        if event["sender"]["login"] == "chromium-wpt-export-bot":
+            all_jobs.discard("wpt-chrome-dev-stability")
+        if event["sender"]["login"] == "moz-wptsync-bot":
+            all_jobs.discard("wpt-firefox-nightly-stability")
+    except KeyError:
+        # Just continue if the username cannot be pulled from the event.
+        logger.info("Unable to read username from event. Continuing.")
```

Comment:
I'm not even sure we need to log this, but if we are maybe make it debug level.

Reply:
I changed it to a debug level log👌